driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -558,7 +564,7 @@ private void retry(final boolean retryCurrent, ConsistencyLevel newConsistencyLe
     }
 
     private void logError(EndPoint endPoint, Throwable exception) {
-      logger.debug("[{}] Error querying {} : {}", id, endPoint, exception.toString());
+      logger.debug("[{}] Error querying {}", id, endPoint, exception);
```

Comment:
I think we can even leave this one as-is, I don't want to spam the debug logs too much for "normal" server error responses.
The error is surfaced via the `errors` map anyway, so it's available if someone wants to inspect it.